codalab/worker/worker_run_state.py

```diff
@@ -439,7 +443,9 @@ def _transition_from_CLEANING_UP(self, run_state):
                     logger.error(traceback.format_exc())
                     time.sleep(1)
 
-        for dep_key, dep in run_state.bundle.dependencies.items():
+        for dep in run_state.bundle.dependencies:
+            dep_asdict = dep._asdict()
+            dep_key = DependencyKey(dep_asdict["parent_uuid"], dep_asdict["parent_path"])
```

Comment:
Why can't you access these fields directly from `dep`?

Reply:
That's true! I have updated the code to access those fields directly from `dep`.